input/fsh/SD_InterfractionVerificationExtension.fsh

```diff
@@ -0,0 +1,47 @@
+// Extension: IntrafractionVerification
+// Id: codexrt-intrafraction-verification
+// Description: "Verification of the patient or target position during a treatment fraction."
+// Title: "Intrafraction Verification"
+// * . ^short = "Intrafraction Verification"
+// * value[x] only CodeableConcept
+// * value[x] from IntrafractionVerificationProcedureVS (extensible)
+// * value[x] 1..1
+
+Extension: IntrafractionVerification
+Id: codexrt-intrafraction-verification
+Description: "Verification of the patient or target position during a treatment fraction."
+Title: "Intrafraction Verification"
+// Add invariant that "the qualifier is only present when the procedure is xray"
+// Need a table with the documentation of the combinations, and clarifying that xray without a qualifer = xray + singular.
+* . ^short = "Intrafraction Verification"
+* insert UsualContexts //Those can be reduced. E.g. I don't expect Obervation(Definition) or Extension as contexts. Maybe remove more.
```

Comment:
Context has to be changed to Contexts1 when merging:
* insert ExtensionContextResource(Procedure)
* insert ExtensionContextResource(ServiceRequest)
* insert ExtensionContextResource(ActivityDefinition)